src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -84,7 +85,22 @@ public void notifyClaimant(
     ) {
         Map<String, String> parameters = aggregateParams(claim, claim.getResponse()
             .orElseThrow(IllegalStateException::new));
-        notify(claim.getSubmitterEmail(), getEmailTemplates().getClaimantResponseIssued(), parameters, reference);
+
+        String emailTemplate = getClaimantEmailTemplate(claim);
+
+        notify(claim.getSubmitterEmail(), emailTemplate, parameters, reference);
+    }
+
+    private String getClaimantEmailTemplate(Claim claim) {
+        if (claim.getResponse().isPresent()) {
+            Response response = claim.getResponse().get();
+
+            if (response.getFreeMediation().isPresent() && response.getFreeMediation().get().equals(YesNoOption.YES)) {
```

Comment:
I think `response.getMediation().orElse(NO)` in a variable before this would be cleaner, then you just check if its yes